src/formats/png/filtering.zig

```diff
@@ -0,0 +1,199 @@
+const std = @import("std");
+const Image = @import("../../Image.zig");
+
+pub const FilterType = enum(u8) {
+    None = 0,
```

Comment:
We use the Zig coding style convention, those enum values needs to be all lowercase

Reply:
This really needs to be in all lowercase